Solaris specific.
/usr/lib/ is default library path and thus it shouldn't be in RPATH and RUNPATH.
-L/-R with /usr/lib are not necessary for Solaris. The linker knows better.

--- pcre-8.38/pcre-config.in_orig	2016-10-05 13:05:50.522096706 +0000
+++ pcre-8.38/pcre-config.in	2016-10-05 13:06:03.384163020 +0000
@@ -34,9 +34,6 @@
 
 libR=
 case `uname -s` in
-  *SunOS*)
-  libR=" -R@libdir@"
-  ;;
   *BSD*)
   libR=" -Wl,-R@libdir@"
   ;;
